### DISCUSSION TITLE
v1.4.0.0 - missing file

### DISCUSSION THREAD

**Discussion by fm-stereo** [amakvana/YuzuModDownloader#31] [General]
I just downloaded the latest version (v1.4.0.0) but upon extracting with Win.rar I noticed it only had the yuzumododnwloader.exe and it did not contain the HtmlAgilityPack.dll

--Does anyone know where I can get the HTMLAgilityPack?
seems like the ModDownloader will not recognize my games and I think it is because I did not copy the agility pack along with the modDownloader to the Yuzu source folder... I have 3 games loaded on Yuzu already yet it says: mods downloaded for 0 games when I run it....Any help would be greatly appreciated.

**Comment by UrsaChrono** [comment]
The HtmlAgilityPack.dll has been bundled with the .exe

**Comment by amakvana** [comment]
As of v1.4.0.0 and onwards, all DLL files are bundled into YuzuModDownloader.exe (and it's Linux variant)

See https://github.com/amakvana/YuzuModDownloader/blob/main/CHANGELOG.md#1400---2023-06-04 for more info
